README.md

```diff
@@ -69,9 +71,11 @@ impl SyncService for HelloServer {
 }
 
 fn main() {
-    let addr = "localhost:10000";
-    HelloServer.listen(addr, server::Options::default()).unwrap();
-    let client = SyncClient::connect(addr, client::Options::default()).unwrap();
+    let reactor = reactor::Core::new().unwrap();
+    let addr = HelloServer.listen("localhost:0".first_socket_addr(),
+                                  server::Options::from(reactor.handle()))
+                          .unwrap();
+    let mut client = SyncClient::connect(addr, client::Options::default().core(reactor)).unwrap();
```

Comment:
This is the easiest way to get up and running in as little code as possible. There's precedence in crates for taking the easy route in examples, for example playing fast and loose with `unwrap`. There's always a balance between "get working quickly" and "the right way", and in this particular case it's not even wrong per se, just quirky.